packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -161,6 +161,7 @@ class InputDateTimePicker extends React.Component {
 				inputRef={ref => {
 					this.inputRef = ref;
 				}}
+				type="date"
```

Comment:
This is not what we want. This will make some browser render the native date picker. We are developing a custom picker, let's remove that and clean the tests and other files that introduce that

Reply:
I am not sure if you are against with the naming `type` itself (which will make some browser render the native date picker) or the approach that I want to render date/time Input conditionally.
For now I renamed `type` to `part`, because I need a prop to flag whether it's the input for date or it's the input for time.